src/OrchardCore.Modules/OrchardCore.Admin/AdminActionModelConvention.cs

```diff
@@ -11,8 +10,7 @@ public class AdminActionModelConvention : IActionModelConvention
     {
         public void Apply(ActionModel action)
         {
-            if (action.Controller.ControllerName.StartsWith("Admin", StringComparison.OrdinalIgnoreCase) ||
-                action.Controller.Attributes.OfType<AdminAttribute>().Any())
+            if (action.Controller.ControllerName == "Admin" || action.Controller.Attributes.OfType<AdminAttribute>().Any())
```

Comment:
Is the controller named "Admin"?

Reply:
Yes, that's the convention. Mvc automatically removes the `Controller` from `AdminController` to set the `ControllerName` property, and I had noted to @jtkech that I made a mistake here, and the OrchardCore convention is to be an admin controller, the controller is either named `AdminController`, or has the `[Admin]` attribute